subsys/bluetooth/mesh/provisionee.c

```diff
@@ -698,8 +698,8 @@ int bt_mesh_prov_enable(bt_mesh_prov_bearer_t bearers)
 		return -EALREADY;
 	}
 
-#if defined(CONFIG_BT_MESH_PROV_DEVICE_LOG_LEVEL)
-	if (CONFIG_BT_MESH_PROV_DEVICE_LOG_LEVEL > 2) {
+#if defined(CONFIG_BT_MESH_PROVISIONEE_LOG_LEVEL)
```

Comment:
We need a follow up task to use the new option in nRF Connect SDK: https://github.com/nrfconnect/sdk-nrf/blob/e29e9d986680d3ddf691f7539b017d2210ae9600/samples/bluetooth/mesh/dfu/distributor/prj.conf#L83

Reply:
I've created a task for it 👍